Trimble.Modus.Components/Converters/ImageNullOrEmptyBoolConverter.cs

```diff
@@ -3,7 +3,8 @@
 public class ImageNullOrEmptyBoolConverter : IValueConverter
 {
     public object Convert(object value, Type targetType, object parameter, System.Globalization.CultureInfo culture)

```

Comment:
Remove the test log

Reply:
Infact, we shouldn't even be needing this class
https://learn.microsoft.com/en-us/dotnet/communitytoolkit/maui/converters/is-not-null-converter